client/app/vps/translations/Messages_fr_FR.xml

```diff
@@ -330,7 +330,7 @@
    <translation id="vps_configuration_veeam_order_step1_loading_error" qtlid="170031">Une erreur est survenue lors du chargement des informations</translation>
    <translation id="vps_configuration_veeam_order_step1_info" qtlid="169329">Pour le premier paiement, vous serez facturé au prorata du nombre de jours restant avant la date de renouvellement de votre VPS.</translation>
    <translation id="vps_configuration_veeam_order_step1_info2" qtlid="170044">Ensuite, le service sera facturé mensuellement au prix de <span class="red bold">{{ price }}</span></translation>
-   <translation id="vps_configuration_veeam_order_step1_question" qtlid="170057">Commande de l'options Backups automatisés sur le VPS <span class="italic normal">{{ serviceName }}</span> ?</translation>
+   <translation id="vps_configuration_veeam_order_step1_question" qtlid="170057">Commande de l'option Backups automatisés sur le VPS <span class="italic normal">{{ serviceName }}</span> ?</translation>
```

Comment:
if a translation is not strictly equals to the previous state, you need to remove qtlid because it is needed to be re-translate.

Reply:
I think we should fix this one in prod, it's clearly in error..

Reply:
You need to remove the qtlid if you want translation team to go over the trad again.  I hope they haven't reproduced a singular vs plural error :)